### ISSUE TITLE
Piano: Crash While Holding Key and Changing Octave

### ISSUE THREAD

**Issue by jsdevv** [SerenityOS/serenity#2466]
```
USERSPACE(45) ASSERTION FAILED: new_octave == m_audio_engine.octave()
/home/jsdevv/src/serenity/Applications/Piano/KnobsWidget.cpp:81
[colonel(0:0)]: Terminating due to signal 6
[FinalizerTask(3:3)]: 0xdeadc0de  _ZN6Kernel9Scheduler10switch_nowEv +64
0xdeadc0de  _ZN6Kernel9Scheduler5yieldEv +85
0xdeadc0de  _ZN6Kernel6Thread30yield_without_holding_big_lockEv +36
0xdeadc0de  _ZN6Kernel7Process11do_killselfEi +191
0xdeadc0de  _ZN6Kernel7Process8sys$killEii +258
0xdeadc0de  syscall_handler +1158
0xdeadc0de  syscall_asm_entry +29
0x080b7214  raise +22
0x080480bd  abort +13
0x080b3d24  tolower +0
0x0804b18a  _ZNK2AK8FunctionIFviEE15CallableWrapperIZN11KnobsWidgetC4ER11AudioEngineR10MainWidgetEUliE_E4callEi +120
0x080b1490  _ZN4Core6Object14dispatch_eventERNS_5EventEPS0_ +74
0x08083858  _ZN3GUI6Window5eventERN4Core5EventE +668
0x080b1490  _ZN4Core6Object14dispatch_eventERNS_5EventEPS0_ +74
0x080ac9ea  _ZN4Core9EventLoop4pumpENS0_8WaitModeE +404
0x080acb65  _ZN4Core9EventLoop4execEv +43
0x08056311  _ZN3GUI11Application4execEv +37
0x0804895b  main +1360
0x0804919e  _start +94

[FinalizerTask(3:3)]: 0xdeadc0de  _ZN6Kernel9Scheduler10switch_nowEv +64
0xdeadc0de  _ZN6Kernel6Thread13die_if_neededEv +118
0xdeadc0de  syscall_handler +1223
0xdeadc0de  syscall_asm_entry +29
0x080af034  _ZN4Core8IODevice5writeEPKhi +24
0x0804cac3  _ZNK2AK8FunctionIFivEE15CallableWrapperIZ4mainEUlvE_E4callEv +177
0x080a9111  _ZZN9LibThread6Thread5startEvENUlPvE_4_FUNES1_ +39
0x080b364b  pthread_create_helper +12
```

**Comment by willmcpherson2** [comment]
I don't think it's due to changing octave while holding a key, but rather due to changing octaves too quickly: #1163

**Comment by willmcpherson2** [comment]
Fixed in #2716
